gadopt/stokes_integrators.py

```diff
@@ -122,35 +123,38 @@ class StokesSolver:
       bcs: Dictionary of identifier-value pairs specifying boundary conditions
       mu: Firedrake function representing dynamic viscosity
       quad_degree: Quadrature degree. Default value is `2p + 1`, where
-                     p is the polynomial degree of the trial space
+                   p is the polynomial degree of the trial space
       cartesian: Whether to use Cartesian coordinates
-      solver_parameters: Dictionary of solver parameters provided to PETSc
+      solver_parameters: Either a dictionary of PETSc solver parameters or a string
+                         specifying a default set of parameters defined in G-ADOPT
       J: Firedrake function representing the Jacobian of the system
       constant_jacobian: Whether the Jacobian of the system is constant
 
     """
-    name = 'Stokes'
+
+    name = "Stokes"
 
     def __init__(
         self,
         z: fd.Function,
         T: fd.Function,
         approximation: BaseApproximation,
-        bcs: Optional[dict[int, dict[str, int | float]]] = None,
-        mu: fd.Function | int | float = 1,
+        bcs: Optional[dict[int, dict[str, Number]]] = None,
+        mu: fd.Function | Number = 1,
```

Comment:
Opening up the solver options to a string definition is nice. But perhaps that should be a seperate pull request? Or am I missing something?

Reply:
I tend to always add commits to the same PR; I just cannot help it. Up to you if you want it as a separate PR. :)

Reply:
Please seperate this solver option change out to a seperate PR.